lists/global.csv

```diff
@@ -1167,6 +1167,9 @@ https://es.wikipedia.org/,CULTR,Culture,2018-07-06,Wikimedia,URLs provided by wi
 https://zh.wikipedia.org/,CULTR,Culture,2018-07-06,Wikimedia,URLs provided by wikipedia based on the most popular language domains per country
 https://ja.wikipedia.org/,CULTR,Culture,2018-07-06,Wikimedia,URLs provided by wikipedia based on the most popular language domains per country
 https://it.wikipedia.org/,CULTR,Culture,2018-07-06,Wikimedia,URLs provided by wikipedia based on the most popular language domains per country
+https://wikimapia.org/,CULTR,Culture,2020-12-12,Jarrousse,
+https://liveuamap.com/,NEWS,News Media,2020-12-12,Jarrousse,
+https://foreignpolicy.com/,NEWS,News Media,2020-12-12,Jarrousse,
```

Comment:
We should drop this site from the local lists to avoid duplicates:
```
lists/ph.csv (line 287): Duplicate URL between Local List and Global List "https://foreignpolicy.com/"
lists/kz.csv (line 296): Duplicate URL between Local List and Global List "https://foreignpolicy.com/"
lists/sy.csv (line 310): Duplicate URL between Local List and Global List "https://foreignpolicy.com/"
lists/cn.csv (line 29): Duplicate URL between Local List and Global List "https://foreignpolicy.com/"
```

Reply:
@hellais I have done this here: https://github.com/citizenlab/test-lists/pull/883